tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestPaths.java

```diff
@@ -122,26 +122,31 @@ public void TestPathHeadersAndValues() {
         CLDRFile englishFile = testInfo.getCldrFactory().make("en", true);
         PathHeader.Factory phf = PathHeader.getFactory(englishFile);
         Status status = new Status();
+        final String exemptLocale = "sv";
+        final String exemptPathIfLocale =
+                "//ldml/dates/calendars/calendar[@type=\"dangi\"]/dateTimeFormats/availableFormats/dateFormatItem[@id=\"yMd\"]";
+
         for (String locale : getLocalesToTest()) {
+            boolean isExemptLocale = locale.equals(exemptLocale);
+
             if (!StandardCodes.isLocaleAtLeastBasic(locale)) {
                 continue;
             }
             CLDRFile file = testInfo.getCLDRFile(locale, true);
             logln("Testing path headers and values for locale => " + locale);
             final Collection<String> extraPaths = file.getExtraPaths();
+
             for (Iterator<String> it = file.iterator(); it.hasNext(); ) {
                 String path = it.next();
-                if (extraPaths.contains(path)) {
+                if (isExemptLocale && path.equals(exemptPathIfLocale)) {
+                    logKnownIssue("CLDR-17544", "Can't reproduce locally");
                     continue;
```

Comment:
Post script: This should have been `if(logKnownIssue(…)) continue;` - only continue if logKnownIssue returns true.